content/akeneo-event-platform/concepts.md

```diff
@@ -0,0 +1,229 @@
+| `name` | Populated by the user at creation | Name of the subscriber                                                          |
+| `subject` | From `X-PIM-URL` header parameter | URL of the targeted source                                                      |
+| `technical_email` | Populated by the user at creation | A contact email will be used to notify you in case of unexpected behavior       |
+| `status` | Automatically populated | The subscriber status                                                           |
+
+The statuses for a subscriber are:
+
+| Status | Description |
+| --- | --- |
+| `active` | The destination will receive notifications for events tracked by the subscriber |
+| `deleted` | The subscriber is inactive and cannot be reactivated |
+
+For comprehensive details on managing subscribers, you can consult the complete API reference [here](/akeneo-event-platform/api-reference.html).
+
+## Subscription
+
+### What is a subscription?
+
+A subscription is an enrollment for one or more event types from a specified source to a particular destination. By adding a new subscription, you ensure that you receive notifications whenever a tracked event occurs in your Akeneo system. Each subscription is linked to a single subscriber.
+
+The following properties represent a subscription:
+
+| Property | Value | Description |
+| --- | --- | --- |
+| `id` | Automatically populated | Identifier of the subscription within the Akeneo Event Platform |
+| `source` | Populated by the user at creation | Source of the event (currently, the only source available is `pim`) |
+| `subject` | From `X-PIM-URL` header parameter | URL of the targeted source |
+| `type` | Populated by the user at creation | Type of the subscription (currently, there are two available types:  `https` and `pubsub`) |
+| `events` | Populated by the user at creation | A list of events that the subscription is tracking |
+| `status` | Automatically populated | The subscription status |
+| `config` | Populated by the user at creation | The subscription configuration, based on the subscription type. See below for further details. |
+
+The statuses for a subscription are:
+
+| Status | Description |
+| --- | --- |
+| `active` | The destination will receive notifications for events monitored by the subscription |
+| `deleted` | The subscription is inactive and cannot be reactivated |
+| `suspended` | The subscription has been deactivated by the event platform itself due to excessive errors, or manually by the user. However, you can resume it. Suspending a subscription stops all events from being sent to it. Events are not saved and are lost until the subscription is resumed |
+| `revoked` | The subscription has been automatically deactivated because the destination responded with an HTTP 404. You can manually reactivate it. Suspending a subscription halts all event deliveries to this subscription. Events are not saved and are lost until the subscription is resumed |
```

Comment:
This status was not designed as it is described here.

A revoked status can only be set by the event platform, and occurs when an App or a Connection  is deleted.
We differentiate the deleted from revoked as "deleted" is an status/action made by the subscriber/user/integrator, the revoked is a event platform decision, to let subscriber/user/integrator understand why we do not receive events anymore.

Reply:
Done on https://github.com/akeneo/pim-api-docs/pull/932/files#